doc/modules/semi_supervised.rst

```diff
@@ -70,24 +70,23 @@ until all samples have labels or no new samples are selected in that iteration.
     .. [1] David Yarowsky. 1995. Unsupervised word sense disambiguation rivaling
        supervised methods. In Proceedings of the 33rd annual meeting on
        Association for Computational Linguistics (ACL '95). Association for
-       Computational Linguistics, Stroudsburg, PA, USA, 189-196. DOI:
-       https://doi.org/10.3115/981658.981684
+       Computational Linguistics, Stroudsburg, PA, USA, 189-196. :doi:`10.3115/981658.981684`
 
 .. _label_propagation:
 
 Label Propagation
 =================
 
 Label propagation denotes a few variations of semi-supervised graph
-inference algorithms. 
+inference algorithms.
```

Comment:
This change is not relevant to the PR. Do you mind reverting it? Thanks!

Reply:
It seems to me that my editor automatically deletes any blanks at the end of a code line... cannot add the blanks for line 82 and 90 simultaneously... Is it ok to just leave it like that?

Reply:
Fine for me... Thanks for clarifying.